department-of-veterans-affairs/va.gov-cms
# VACMS-13806: hide counters on text fields when editing translations.

## Description

#13806 

## Testing done
See QA steps; there are also automated tests.

## QA steps

1. Create a Landing Page node: https://pr14043-qnm2i2mxdenkoiueccfejlawjv3buksa.ci.cms.va.gov/node/add/basic_landing_page, filling out the required fields. Save it.
2. Edit the node. Attempt to add too much text to Page Introduction (over 1000 characters).
  - [x] You should see an error on the field if you attempt to submit it with too many characters.
3. Click 'Translate' to get a list of the three languages we support. In the 'Spanish' row, click 'add'.
  - [ ] Verify that you do not see any counters on this page on the text fields, where they were present on English.
4. Add over 1000 characters into Page Introduction. Fill out anything else necessary and save.
  - [x] You should be allowed to save and not prevented from submitting.

### Definition of Done


### Select Team for PR review
